docs/spec.md

```diff
@@ -252,3 +252,83 @@ const decoded = JSON.parse(new TextDecoder().decode(decrypted.plaintext));
 }
 */
 ```
+
+## Use Case Examples
+
+### Using SHL to share an interactive experience
+
+While the SMART Health Links spec focuses on providing access to structured data, it's often
+useful to share an interactive experience such as a web-based diagnostic portal where the
+SHL Data Recipient can review and add comments to a patient record. This can be accomplished
+in SHL with a manifest entry of type `application/fhir+json` that provides a
+[FHIR Endpoint resource](https://hl7.org/fhir/endpoint.html) where:
+
+* `name` describes the interactive experience with sufficient detail for the SHL Recipient to decide whether to engage
+* `connectionType` is `{"system": "https://smarthealthit.org", "code": "shl-interactive-experience"}`
+* `address` is the URI for the interactive experience
+* `period` optionally documents the window of time when the interactive experience is available
+
+For example, the manifest for an SHL that offers the user the opportunity to "Review a case"
+might include a `application/fhir+json` entry with:
+
+```json
+{
+  "resourceType": "Endpoint",
+  "status": "active",
+  "name": "Review and comment on Alice's case in ACME Medical Diagnostic Portal",
+  "address": "https://interact.example.org/case-id/521039c3-4bb9-45bd-8271-6001d2f4dea9",
+  "period": {"end": "2022-10-20T12:30:00Z"},
+  "connectionType": {"system": "https://smarthealthit.org", "code": "shl-interactive-experience"},
+  "payloadType": [{"system": "http://terminology.hl7.org/CodeSystem/endpoint-payload-type", "code": "none"}],
+}
+```
+
+Notes:
+
+* There is no perfect FHIR resource for documenting an interactive experience URL. `Endpoint` and
```

Comment:
Should this bullet be under the "Upgrading from SHL to a consumer-mediated SoF connection" ?